### PR TITLE
Image Upload and Validation

### PR DESCRIPTION
**Pull request by Vinitha-2605**

### CODE REVIEW

File: ImageView/models.py [hunk 1]
```diff
@@ -8,12 +9,17 @@ class Category(models.Model):
     def __str__(self):
         return self.category
 
+def validate_image(image):
+         img = PIL.Image.open(image)
+         if img.size[0] > 1000:
```
Comment (rajmohan-ideas2it) [inline ImageView/models.py]: We need the memory size of the image. Why do we need to validate the size of the image?
What a are you trying to validate?
image

File: ImageView/models.py [hunk 2]
```diff
@@ -8,12 +9,17 @@ class Category(models.Model):
...
+            raise ValidationError("Image dimensions must not exceed 1000x1000 pixels.")
+     
 class Image(models.Model):
     Title = models.CharField(max_length=20)
```
Comment (rajmohan-ideas2it) [inline ImageView/models.py:20]: move hardcoded values to config files 
eg
max_length = config.img_max_lenght

File: ImageView/views.py [hunk 3]
```diff
@@ -49,19 +54,24 @@ def imagedetail(request, id):
...
-     image = Image.objects.all()
-     return render(request, "searchimages.html",{
-        "images": image,
+   form = Category.objects.values()
+   image = Image.objects.all().values()
+   data = list()
+   for imagedata in image:
+      if imagedata['Imagedata'] is not None:
+       imagedata['Imagedata'] = base64.b64encode(imagedata['Imagedata']).decode('utf-8')
+       print( imagedata['Imagedata'])
```
Comment (rajmohan-ideas2it) [inline ImageView/views.py]: Dont use print,use loggers

**Review by Vinitha-2605 — COMMENTED** [review]
Resolved the comments

File: ImageView/models.py (hunk 1 above)
Comment (Vinitha-2605) [reply ImageView/models.py]: Validated the image width and height size.

File: ImageView/models.py (hunk 2 above)
Comment (Vinitha-2605) [reply ImageView/models.py:20]: Created the config file for hardcoded values

File: ImageView/views.py (hunk 3 above)
Comment (Vinitha-2605) [reply ImageView/views.py]: Removed the print statement

File: ImageView/views.py [hunk 4]
```diff
@@ -2,11 +2,13 @@
 from django.http import HttpResponse
 from django.db.models import Q
 import base64
+import logging
 
 from .forms import ImageForm, CategoryForm
 from .models import Image, Category
 
 # Create your views here.
+logging.basicConfig(level=logging.INFO)
```
Comment (rajmohan-ideas2it) [inline ImageView/views.py:11]: Use separate logger file and import in anywhere

**Review by rajmohan-ideas2it — APPROVED** [review]

**Pull request merged**